### ISSUE TITLE
There are different paths to submobules in the repo and in configure.ac

### ISSUE THREAD

**Issue by qinidema** [4n6ist/bulk_extractor-rec#3]
**Subject of the issue**
The paths of your new submodules (in the root of the repo) and paths in configure.ac (src/$submobule) file are different. And so, the build fails (at least on Linux).

**Steps to reproduce**
As per the readme file:
1. Run `git clone --recursive https://github.com/4n6ist/bulk_extractor-rec.git`
2. Run `cd bulk_extractor-rec`
3. Run `sh bootstrap.sh`

**Expected behaviour**
Configure script is generated

**Actual behaviour**
```
[user@box] ~/bulk_extractor-rec $ sh bootstrap.sh
bringing in submodules
next time check out with git clone --recursive
bringing in submodules
next time check out with git clone --recursive
/usr/bin/m4:configure.ac:130: cannot open `src/dfxml/src/dfxml_configure.m4': No such file or directory
/usr/bin/m4:configure.ac:131: cannot open `src/be13_api/be13_configure.m4': No such file or directory
autom4te: /usr/bin/m4 failed with exit status: 1
autoheader: '/usr/bin/autom4te' failed with exit status: 1
aclocal: error: configure.ac:130: file 'src/dfxml/src/dfxml_configure.m4' does not exist
/usr/bin/m4:configure.ac:130: cannot open `src/dfxml/src/dfxml_configure.m4': No such file or directory
/usr/bin/m4:configure.ac:131: cannot open `src/be13_api/be13_configure.m4': No such file or directory
autom4te: /usr/bin/m4 failed with exit status: 1
/usr/bin/m4:configure.ac:130: cannot open `src/dfxml/src/dfxml_configure.m4': No such file or directory
/usr/bin/m4:configure.ac:131: cannot open `src/be13_api/be13_configure.m4': No such file or directory
autom4te: /usr/bin/m4 failed with exit status: 1
automake: error: autoconf failed with exit status: 1

The bootstrap.sh is complete. Be sure to run ./configure.
[user@box] ~/bulk_extractor-rec $ ./configure
-bash: ./configure: No such file or directory
```
(ofcourse no ./configure is present after the errors)

**Dirty fix**
```
[user@box] ~/bulk_extractor-rec $ ln -s ../dfxml/ ./src/dfxml
[user@box] ~/bulk_extractor-rec $ ln -s ../be13_api/ ./src/be13_api
```
After that bootstrap.sh works, and configure script is generated successfully.

**Comment by 4n6ist** [comment]
Thanks! I forgot to check it after I updated to latest submodule. I will find reasonable solution.

**Comment by simsong** [comment]
(FYI — The master fork at https://github.com/simsong/bulk_extractor now includes the record carving modules and shouldn't have this problem.)

**Comment by qinidema** [comment]
Thank you, @4n6ist! :)
Thank you @simsong! :) I've noticed that already, but unfortunately the "main" bulk_extractor is also difficult to build nowadays cause of https://github.com/simsong/bulk_extractor/issues/126

**Comment by simsong** [comment]
Huh, I thought I fixed that. Thanks for the reminder. Will do shortly.

**Comment by 4n6ist** [comment]
Fixed at commit 5a03a18c, and succeeded to build without error at 1964395
